src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,58 @@
+.calendar {
+  position: relative;
+  max-width: getWidth($distans-between-cell, $cell-size, $field-padding);
+
+  &__field {
+    display: flex;
+    flex-wrap: wrap;
+    gap: $distans-between-cell;
+    padding: 50px $field-padding;
+
+    @for $index1 from 28 through 31 {
+      &--mounth-length--#{$index1} :nth-child(n + #{$index1 + 1}) {
+        display: none;
+      }
+    }
+  }
+
+  &__cell {
+    @include square($cell-size);
+
+    display: flex;
+    justify-content: center;
+    box-sizing: border-box;
+    border: 1px solid #000;
+    background-color: #eee;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    transition: background-color 0.5s, transform 0.5s;
+
+    @for $index from 1 through $cell-count {
+      &--content-#{$index}::before {
+        content: "#{$index}";
+        align-self: center;
+      }
+    }
+
+    $days: 
+      "mon" 0,
+      "tue" 101px,
+      "wed" 202px,
+      "thy" 303px,
+      "fri" 404px,
+      "sat" 505px,
+      "sun" 606px;
```

Comment:
looks like as `magic` numbers. create variable for `gap` and multiply to `cell size` here or in loop.